test.py

```diff
@@ -814,6 +814,9 @@ def _reset(self) -> None:
         old_tmpfile = pathlib.Path(self.tmpfile)
         if old_tmpfile.exists():
             old_tmpfile.unlink()
+
+
+    def _prepare_args(self, options: argparse.Namespace):
```

Comment:
This code was previously part of `_reset` -- so don't we need to call it at the end of `_reset` now?

Reply:
Ah, you don't have access to `options` in `_reset`

Reply:
Yes, it was the main reason. Putting this method after `_reset` and before we start execution will be OK, but I guess better place will be in `__init__` after reset. So actually, it will be the same order of execution as it was before.